src/setup-docker/index.ts

```diff
@@ -101,9 +101,9 @@ export default class SetupDockerGenerator extends Generator {
     });
   }
 
-  _createConnectionProfiles(global: Global, orgsTransformed: OrgConfig[]): void {
+  _createConnectionProfiles(global: Global, orgsTransformed: OrgConfig[], channels: ChannelConfig[] = [], ordererGroups: OrdererGroup[] = []): void {
```

Comment:
please require explicit parameters (less prone to errors)
```suggestion
  _createConnectionProfiles(global: Global, orgsTransformed: OrgConfig[], channels: ChannelConfig[], ordererGroups: OrdererGroup[]): void {
```